### DISCUSSION TITLE
how to write psi bar in feyncalc?

### DISCUSSION THREAD

**Discussion by rahaman3921** [FeynCalc/feyncalc#190] [General]
I want to write QuantumField [SpiBar(Ad joint spinor)] .how can I write it?

**Comment by rahaman3921** [comment]
Find it...Thanks
Answer
QuantumField[OverBar[/[psi]]

**Reply by vsht** [reply]
`QuantumField[AntiQuarkField]` works as well.

**Reply by rahaman3921** [reply]
Ok....Thank you
